src/helm/benchmark/run_specs.py

```diff
@@ -620,6 +620,14 @@ def get_cleva_machine_translation_metric_specs() -> List[MetricSpec]:
     ] + get_basic_metric_specs([])
 
 
+def get_cleva_paraphrase_generation_metric_specs() -> List[MetricSpec]:
+    return [
+        MetricSpec(
+            class_name="helm.benchmark.metrics.paraphrase_generation_metrics.CLEVAParaphraseGenerationMetric", args={}
```

Comment:
I will expect we can pass `alpha=0.8` from `args` and arguments in `get_cleva_paraphrase_generation_metric_specs` here.